.github/workflows/ci-windows.yml

```diff
@@ -7,7 +7,47 @@ permissions:
 
 jobs:
   Windows:
-    name: Windows
+    name: Windows (Native OPAM)
+    runs-on: windows-latest
+    steps:
+      - name: 🔭 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 😈 Evil hacks
+        run: rm rocq-prover.opam
+
+      - name: 😈😈 Evil hack 2
+        run: rm test-suite/success/rewrule.v
+
+      - name: 😈😈😈 Evil hack 3
+        run: rm test-suite/failure/uip_loop.v
+
+      - name: 😈😈😈😈 Evil hack 4
+        run: rm rocq-doc.opam
+
+      - name: 🐫 Setup OCaml
+        uses: ocaml/setup-ocaml@v3
+        with:
+          ocaml-compiler: 4.14
+          dune-cache: true
+          opam-local-packages: |
+            coqide-server.opam
+            rocq-*.opam
+
+      - name: 🐫🐪🐫 Get Rocq dependencies
+        run: opam install --deps-only ./coqide-server.opam ./rocq-*.opam
```

Comment:
if we use an explicit list of opam files instead of a global here (as in the gitlab CI) we wouldn't need to `rm` the ones we don't want.

Reply:
also since this is --deps-only it may make sense to only list rocq-runtime and rocq-devtools, since the others are not supposed to have ocaml deps

Reply:
I see your point, however we have a bit of debate on what we want, some people wants to do `opam install .` and have it work, so I dunno, the above tries to test more stuff so we can detect breaking changes. Happy to tweak it as you see fit.

Reply:
We don't rm anymore since rocq-doc was removed, so it's fine like this